torch/_functorch/_aot_autograd/runtime_wrappers.py

```diff
@@ -1344,12 +1344,12 @@ def _same_dtype_views(view1, view2):
     storage_ref_to_idx: dict[StorageWeakRef, list[int]] = collections.defaultdict(list)
     base_args = []
     other_args = []
-    for i, inpt in enumerate(fwd_inputs):
-        if isinstance(inpt, Tensor):
-            storage_ref = StorageWeakRef(inpt.untyped_storage())
+    for i, input in enumerate(fwd_inputs):
```

Comment:
These are not really ok either. We shouldn't override the builtin input wherever possible